common/meta/raft-store/tests/it/state_machine/mod.rs

```diff
@@ -38,6 +38,8 @@ use pretty_assertions::assert_eq;
 use crate::init_raft_store_ut;
 use crate::testing::new_raft_test_context;
 
+#[cfg(feature = "mock_utc")]
+mod mock;
```

Comment:
It's a bit wired to have a mock mod in non-testing code.

In fact, we are building a `wrapper` with mock support here. How about implementing a `common/time` crate and add mock support internally?